### PR TITLE
tests: add kill lightning test in checkpoint_chunks case

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 
add test case to cover lightning is killed durning data import

### What is changed and how it works?
use gofail to control lightning to kill itself after one chunk is imported

### Check List 

Tests 

 - Integration test

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @kennytm @lonng

File: tests/checkpoint_chunks/run.sh [hunk 1]
```diff
@@ -47,21 +59,43 @@ for i in $(seq "$CHUNK_COUNT"); do
...
+# If checkpoint does work, this should only kill $CHUNK_COUNT instances of lightnings.
+export GOFAIL_FAILPOINTS='github.com/pingcap/tidb-lightning/lightning/restore/KillIfImportedChunk=return'
+
+for i in $(seq "$CHUNK_COUNT"); do
+    echo "******** Importing Chunk Now (step $i/$CHUNK_COUNT) ********"
+    run_lightning
+done
+
+set +e
+while true; do
```
Comment (kennytm) [inline tests/checkpoint_chunks/run.sh]: Add a retry limit to avoid getting stuck in infinite loop.

**Pull request merged**